File: dataset/output/CLAUDE_claude-3-5-sonnet-20240620/Task146_CLAUDE_claude-3-5-sonnet-20240620.c

#include <stdio.h>

int findMissingNumber(int arr[], int size) {
    int n = size + 1;
    int expectedSum = n * (n + 1) / 2;
    int actualSum = 0;
    for (int i = 0; i < size; i++) {
        actualSum += arr[i];
    }
    return expectedSum - actualSum;
}

int main() {
    int inputArray[] = {1, 2, 4, 6, 3, 7, 8};
    int size = sizeof(inputArray) / sizeof(inputArray[0]);
    int missingNumber = findMissingNumber(inputArray, size);
    printf("The missing number is: %d\\n", missingNumber);
    return 0;
}

File: dataset/output/CLAUDE_claude-3-5-sonnet-20240620/task146/Task146_CLAUDE_claude_3_5_sonnet_20240620.java

import java.util.Arrays;

public class Task146_CLAUDE_claude_3_5_sonnet_20240620 {
    public static int findMissingNumber(int[] arr) {
        int n = arr.length + 1;
        int expectedSum = n * (n + 1) / 2;
        int actualSum = Arrays.stream(arr).sum();
        return expectedSum - actualSum;
    }

    public static void main(String[] args) {
        int[] inputArray = {1, 2, 4, 6, 3, 7, 8};
        int missingNumber = findMissingNumber(inputArray);
        System.out.println("The missing number is: " + missingNumber);
    }
}
